documentatie/functionaliteit/opties.md

```diff
@@ -0,0 +1,20 @@
+
+# Lijst met opties voor Abacus
+
+Deze lijst heeft tot doel het verzamelen van opties en mogelijkheden voor Abacus, bijvoorbeeld vanuit tests en gesprekken met gebruikers.
+
+## Gebruikersinterface
+
+### Coordinatorscherm
+
+- Sorteren op stembureaunummer
+- Numerieke aantallen per stap anders sorteren: fouten en meldingen bovenaan, verder op volgorde van het werkproces
```

Comment:
Over welke pagina gaat dit?

Reply:
Gaat het hier dan puur/alleen om het naar boven verplaatsen van 'werkvoorraad'? Dat is de enige de afwijkt van de volgorde van het werkproces. En een bewuste keuze omdat de volgorde in dit overzicht overeenkomt met de balkjes in de voortgangsbalk en de tabellen in de rechterkolom.